src/components/AddressSearch/index.tsx

```diff
@@ -414,10 +439,18 @@ function AddressSearch(
                         }}
                         styles={{
                             textInputContainer: [styles.flexColumn],
-                            listView: [StyleUtils.getGoogleListViewStyle(displayListViewBorder), styles.overflowAuto, styles.borderLeft, styles.borderRight, !isFocused && {height: 0}],
+                            listView: [
+                                StyleUtils.getGoogleListViewStyle(displayListViewBorder),
+                                listViewOverflow,
```

Comment:
Given the recent changes, is this style still needed?

Reply:
@eVoloshchak good point! I just removed this 😄 Thanks!